$NetBSD$

Back out broken change made between 4.1.0 and 4.1.1 as wont work on non Linux.
Will need to be redone properly.

--- ark/plugins/libzipplugin/zipplugin.cpp.orig	2008-09-02 22:50:57.000000000 +1200
+++ ark/plugins/libzipplugin/zipplugin.cpp
@@ -30,8 +30,8 @@
 //usually compiled with off_t to 32bit. in the long run libzip should be
 //compiled according to bugs.kde.org bug #167018
 //but for now this will probably do
-#define __off_t_defined
-typedef quint32 off_t;
+//#define __off_t_defined
+//typedef quint32 off_t;
 
 #include "kerfuffle/archiveinterface.h"
 #include "kerfuffle/archivefactory.h"
